recipe/conda_build_config.yaml

```
@@ -46,8 +46,9 @@ blas_impl:
   - mkl                         # [x86_64]
   - generic
 
+# https://github.com/conda-forge/.cirun
 github_actions_labels:          # [linux]
-- cirun-openstack-gpu-2xlarge   # [linux]
+- cirun-openstack-cpu-2xlarge   # [linux]
```

I guess you wanted this?

```suggestion
- cirun-openstack-gpu-2xlarge   # [linux]
```